lms/djangoapps/badges/api/tests.py

```diff
@@ -0,0 +1,280 @@
+"""
+Tests for the badges API views.
+"""
+from django.conf import settings
+from django.test.utils import override_settings
+
+from badges.tests.factories import BadgeAssertionFactory, BadgeClassFactory, RandomBadgeClassFactory
+from util.testing import UrlResetMixin
+from xmodule.modulestore.tests.factories import CourseFactory
+from mobile_api.testutils import MobileAPITestCase, MobileAuthUserTestMixin
+
+
+FEATURES_WITH_BADGES_ENABLED = settings.FEATURES.copy()
+FEATURES_WITH_BADGES_ENABLED['ENABLE_OPENBADGES'] = True
```

Comment:
The standard way of overriding a feature flag in unit tests is by decorating the test with:
```
@patch.dict(settings.FEATURES, {'ENABLE_OPENBADGES': True})
```